app/views/posts/index.html.erb

```diff
@@ -1,8 +1,8 @@
 <section>
 <div class = 'the-user'>
-    <div class = 'users-photo'>
-        <img src='<%= @user.photo %>' alt = '<%= @user.name %> photo' />
-    </div>
+    <%# <div class = 'users-photo'>
+    <img src='<%= @user.photo ' alt = '<%= @user.name  photo' />
+    </div> %>
```

Comment:
- [ ] Kindly remove this commented code. It was not present during previous review.